addons/mail/models/mail_channel.py

```diff
@@ -540,21 +540,48 @@ def _channel_message_notifications(self, message):
                 notifications.append([channel.uuid, dict(message_values)])
         return notifications
 
+    @api.model
+    def partner_info(self, all_partners, direct_partners):
+        """
+        Return the information needed by channel to display channel members
+            :param partners: list of res.parner():
+            :param partners: list of res.parner():
+            :returns: a list of {id', 'name', 'email'} for each partner and adds {im_status, out_of_office_message} for direct_partners.
```

Comment:
missing `id'` => `'id'` (missing commas), and same for optional keys for `direct_partnners`